frontend/public/locales/fr/translation.json

```diff
@@ -625,6 +635,17 @@
     }
   },
   "noVideoCorrespondsToSearchCriterias": "Aucune vidéo ne correspond à vos critères de recherche.",
+  "preferences": {
+    "errorOccurredWhileRetrievingPreferences": "Désolé, une erreur est survenue, impossible de récupérer vos préférences.",
+    "preferences": "Préférences",
+    "preferencesOfThePoll": "Préférences du scrutin :"
```

Comment:
It seems to be the first time that we use the word "scrutin" in the context of video comparisons. 

Maybe we could stick to a more generic wording like "Préférences concernant {pollName}" ?

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1501/commits/ab1b0ead398c986fab8854d74271706eab1df673